scripts/start-dev.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)